modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -156,11 +156,11 @@ class EditorCommands {
   }
 
   /**
-   * Queries the command value for example the current fontsize.
+   * Queries the command value. For example: The current fontsize.
    *
    * @method queryCommandValue
    * @param {String} command Command to check the value of.
-   * @return {Object} Command value of false if it's not found.
+   * @return {String} Command value of false (`""`) if the command value is not found.
```

Comment:
Saying this returns false is odd... Maybe say it "returns an empty string if the query command is not found."? Also I think there was previously a typo here, as I believe that should be "Command value **or**".

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5849/commits/c7517c56161ed950eeaaeeb265ffcb96f4c2fb5a